### PR TITLE
Update GitHub insight

### PR DESCRIPTION
**Pull request by Irma12**
## Hey, I just made a Pull Request!

Updated github insights version

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by dtuite** [comment]
Hi @Irma12 . The backstage repository require a DCO signature because it's a CNCF project. If you click the "Details" link next to DCO in the GitHub checks it will take you to [this page](https://github.com/backstage/backstage/pull/5616/checks?check_run_id=2528306892) which explains what to do.

[More info here](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin)

File: yarn.lock [hunk 1]
```diff
@@ -3715,7 +3744,25 @@
     react-is "^16.8.0 || ^17.0.0"
     react-transition-group "^4.4.0"
 
-"@material-ui/icons@^4.9.1":
+"@material-ui/core@^4.11.3", "@material-ui/core@^4.11.4":
```
[the remark this answers is not in the document]
Comment (Irma12) [reply yarn.lock:3747]: Hi @jhaals, thank you for looking at this. Would you consider updating core libraries versions if I add separate PR for updating those dependencies? Or is there some specific reason, I should keep in mind, for keeping exact versions you guys are using now? :)

[the remark this answers is not in the document]
Comment (jhaals) [reply yarn.lock:3747]: To my knowledge I think there is a reason to keep any of those dependencies locked, except for major versions but if they comply with semver it should be fine. To me a separate PRs for bumping the dependencies sound like a good idea. I'll let @benjdlambert or @freben reply to if there are reasons or things to keep in mind before proceeding :)

[the remark this answers is not in the document]
Comment (Xantier) [reply yarn.lock:3747]: @jhaals @freben @benjdlambert Do you have a list internally of these core libs that are treated in a singletony way in Backstage. We have a bunch of plugins so would be good to keep track these and their versions somehow.

[the remark this answers is not in the document]
Comment (Irma12) [reply yarn.lock]: I have created a separate PR targeting only plugin updates https://github.com/backstage/backstage/pull/5655 in order to ease out a process of review. However as @Xantier stated we would be super grateful if you could share a list of the core libs we should keep in mind, so I will keep this PR opened for that matter:)

[the remark this answers is not in the document]
Comment (freben) [reply yarn.lock]: In general I think you want to look at the yarn.lock when submitting a PR. If it contains duplicates of significant libraries, _especially_ the backstage ones themselves, you may have to ask yourselves if you want to align some version somewhere. It's never healthy to stack more and more duplicates of libraries in there. So in part, it's based on gut feeling and general code hygiene, and in part it's based on never ever allowing multiple core backstage libraries at the same time.

**Comment by freben** [comment]
I gave a bit of feedback in the discussion thread above. If it's satisfactory, maybe close this one?

**Pull request closed without merging**